nautobot/utilities/tests/test_filters.py

```diff
@@ -1020,28 +1097,28 @@ def get_filterset_count(self, params, klass=None):
     def test_default_icontains(self):
         """Test a default search for an "icontains" value."""
         params = {"q": "Test Site"}
-        self.assertEqual(self.get_filterset_count(params), 4)
+        self.assertEqual(self.get_filterset_count(params), self.queryset.filter(name__icontains="Test Site").count())
         params = {"q": "Test Site 3"}
-        self.assertEqual(self.get_filterset_count(params), 1)
+        self.assertEqual(self.get_filterset_count(params), self.queryset.filter(name__icontains="Test Site 3").count())
         # Trailing space should only match the first 3.
         params = {"q": "Test Site "}
-        self.assertEqual(self.get_filterset_count(params), 3)
+        self.assertEqual(self.get_filterset_count(params), self.queryset.filter(name__icontains="Test Site ").count())
 
     def test_default_exact(self):
         """Test a default search for an "exact" value."""
         params = {"q": "1234"}
-        self.assertEqual(self.get_filterset_count(params), 1)
-        params = {"q": "123"}
-        self.assertEqual(self.get_filterset_count(params), 0)
+        self.assertEqual(self.get_filterset_count(params), self.queryset.filter(asn__exact="1234").count())
+        params = {"q": "581"}
```

Comment:
changed the testcase here because "123" is contained in one of the site's shipping address which is not default to exact filtering.